### PR TITLE
linux: add jdk-20+36

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: linux/Jenkinsfile [hunk 1]
```diff
@@ -362,14 +370,22 @@ def uploadRpmArtifacts(String DISTRO, String rpmArch) {
...
         'ppc64le': 'ppc64le',
         's390x'  : 's390x'
     ]
+    // if VERSION is 8 or 20 remove s390x from the list
+    if (VERSION == '8' || VERSION == '20') {
+        rpmArchList.remove('s390x')
+    }
+    // if VERSION is 20 remove armv7l from the list
+    if (VERSION == '20') {
+        rpmArchList.remove('armv7l')
```
Comment (zdtsw) [inline linux/Jenkinsfile:386]: armv7hl or armv7l ?

**Review by tellison — APPROVED** [review]
Noted that s390x & armv7hl are removed from buildVersion 20

File: linux/jdk/redhat/src/main/packaging/temurin/20/temurin-20-jdk.spec [hunk 2]
```diff
@@ -123,15 +99,9 @@ Source1: %{source_url_base}/jdk-%{upstream_version_url}/OpenJDK20U-jdk_%{vers_ar
 # Second architecture (ppc64le)
 Source2: %{source_url_base}/jdk-%{upstream_version_url}/OpenJDK20U-jdk_%{vers_arch2}_linux_hotspot_%{upstream_version_no_plus}.tar.gz
 Source3: %{source_url_base}/jdk-%{upstream_version_url}/OpenJDK20U-jdk_%{vers_arch2}_linux_hotspot_%{upstream_version_no_plus}.tar.gz.sha256.txt
-# Third architecture (s390x)
+# Fourth architecture (aarch64)
```
Comment (zdtsw) [inline linux/jdk/redhat/src/main/packaging/temurin/20/temurin-20-jdk.spec:102]: Third

**Review by smlambert — APPROVED** [review]

**Comment by jerboaa** [comment]
Any particular reason why s390x is not on the list?

**Review by zdtsw — APPROVED** [review]

**Comment by tellison** [comment]
> Any particular reason why s390x is not on the list?

I believe that it is unavailable at the moment

**Comment by sxa** [comment]
> > Any particular reason why s390x is not on the list?
> 
> I believe that it is unavailable at the moment

Correct, along with arm32

**Pull request merged**